whylog/assistant/regex_assistant_exceptions.py

```diff
@@ -0,0 +1,7 @@
+class NoDateGroupException(Exception):
+    def __init__(self, line):
+        self.line = line
+        self.message = 'No date group in regex'
```

Comment:
Line number probably won't say user nothing because it is Assistant's and Front's internally indexation. This indexation could be even not visible to user. So I am not sure if it should be in message.